javascript-loops-practice/filter-out-nulls.js

```diff
@@ -0,0 +1,12 @@
+/* exported filterOutNulls */
+function filterOutNulls(values) {
+  var newArr = [];
+  for (var i = 0; i < values.length; i++) {
+    if (values[i] === null) {
+      delete values[i];
```

Comment:
Nooooooo, this will change the original array's value 😭 

If it is null, just ignore it, don't delete it.